### PR TITLE
Colour tokens

### PR DESCRIPTION
**Pull request by anttio**
Initial setup of semantic colour naming scheme to be used across different platforms.

### CODE REVIEW

File: tokens/colours.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Colours
+
+Neptune uses semantic colour naming scheme to describe what it is than what it looks like.
```
Comment (gusso) [inline tokens/colours.md]:
```suggestion
Neptune uses a semantic colour naming scheme to describe what it is rather than what it looks like.
```

**Comment by gusso** [comment]
Absolutely love the simplicity of this. I think we can probably get away with removing a few of them still, such as using `text normal` for inputs, but we can review this when we audit forms.

File: tokens/colours.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| --------------------- | ------------------------------ |
+|  **Core**             |                                |
+| `primary`             |                                |
+| `accent`              |                                |
+| `positive`            |                                |
+| `negative`            |                                |
+| `warning`             |                                |
+|                       |                                |
+|  **Text**             |                                |
+| `text primary`        | Headlines and bold text        |
```
Comment (giljulio) [inline tokens/colours.md]: On android we use `text important` for bold text

File: tokens/colours.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+| `text normal`         | Body text                      |
+| `text important `     | Highlights                     |
+| `text minor`          | Small print                    |
```
Comment (giljulio) [inline tokens/colours.md]: Should we keep this? I don't think we got such a thing on mobile

Comment (anttio) [reply tokens/colours.md:18]: On web this is currently used in few places. So I would keep this for now, and see if we would be able to remove it in the future.

**Pull request merged**